Module4_Labs/Lab7_Zoologist/Cards/212.md

```diff
@@ -26,18 +26,22 @@ Here we print out the prompt for the user and set up the **input** to be taken i
     x=input("")
 ```
 
-Following that are our **if-else** statements that are written according to the prompt. `If` any of these if statements are satisfied then the `while` loop is terminated as the **boolean** statement is **False**.
+Following that are our **if-else** statements that are written according to the prompt. `If` any of these if statements are satisfied then we will set the boolean variable `invalidInput` to False, this means that our input is valid and therefore we will terminate from the while loop. If any of the prompt is not satisfied then the `invalidInput` variable remains True and we will repeat the loop for user prompt again.
 
 ```python
-    if(x=="1"or x=="bubble"):
-        bubbleSort(ranks.copy())
-        invalidInput=False
-    elif(x=="2"or x=="insertion"):
-        insertionSort(ranks.copy())
-        invalidInput=False
-    elif(x=="3"or x=="selection"):
-        selectionSort(ranks.copy())
-        invalidInput=False
-    else:
-        print("Not a valid input, try again.")
+    while(invalidInput):
```

Comment:
can you please explain why the if-else statements are the way they are? remind them what "valid" inputs looks like, and that those are what they should be checking against